app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -793,6 +793,11 @@ def prov_set_show_vars
         end
         @options[:wf] = options[:wf]
       end
+    elsif @miq_request.resource_type == 'ServiceRetireRequest'
+      @view, @pages = nil
+      if (service_id = @miq_request.options[:src_ids].first) && (service = Service.find_by(:id => service_id)) && Rbac.filtered_object(service)
+        @view, @pages = get_view(Vm, :parent => service, :view_suffix => "VmReconfigureRequest")
```

Comment:
I agree that this is a bug, I am however unsure of why VmReconfigureRequests would come into play here, as @tinaafitz pointed out.

Reply:
AFAIK it's only a matter of what columns are displayed for each Vm. Here I reused the reconfigure view which shows CPU and memory resources consumed by each Vm.

But let me double check on that.

Reply:
@d-m-u just realized view_suffix has to do with Rbac view, not the haml views. I do apologize for screwing this up, did it right this time. Thanks for pointing out!